apps/vscode-extension/src/preview/components/MarkdownPreview.tsx

```
@@ -0,0 +1,380 @@
+        <div
+          className='slide__container absolute top-[50%] left-[50%] w-[960px] h-[540px] transition-transform duration-300'
+          style={{
+            transform: `translate(-50%, -50%) scale(${isZoomed ? scale * zoomLevel : 'var(--demotime-scale, 1)'}) translate(${isZoomed ? panOffset.x / (scale * zoomLevel) : 0}px, ${isZoomed ? panOffset.y / (scale * zoomLevel) : 0}px)`
+          }}>
+          <div
+            ref={slideRef}
+            className={`slide__layout ${layout || "default"} ${transition || ""}`}
+            style={getBgStyles()}>
+            {
+              header && (
+                <header className={`slide__header z-20`} dangerouslySetInnerHTML={{ __html: header }}></header>
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)